lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -422,6 +428,7 @@ def execute
         end
         return HammerCLI::EX_OK
       end
+      # rubocop:enable Metrics/AbcSize
```

Comment:
If there is an easy way to avoid disabling the cops by breaking up the method, that would be better IMO, but if it makes things more convoluted and less readable, I think its fine to disable the cop. (I'm not sure how strict this repo is about things like this)

Reply:
This method is pretty scary looking IMO :)

Reply:
After talking with @akofink I made a redmine issue to refactor the execute method after the 6.5 beta stuff is in. Here is the the redmine to track it.

https://projects.theforeman.org/issues/25931

Reply:
Added to Kanboard as well.